Configuration/StandardSequences/python/Reconstruction_cff.py

```diff
@@ -65,12 +66,12 @@
 phase2_timing_layer.toReplaceWith(localreco_HcalNZS,_phase2_timing_layer_localreco_HcalNZS)
 
 _ctpps_2016_localreco = localreco.copy()
-_ctpps_2016_localreco += totemRPLocalReconstruction
+_ctpps_2016_localreco += totemRPLocalReconstruction+ctppsDiamondLocalReconstruction
```

Comment:
Now using the recoCTPPS object. @slava77, do you suggest to rename it in this PR, or open a dedicated PR for this?